docs/proposals/20191030-machine-health-checking.md

```diff
@@ -158,10 +164,11 @@ type target struct {
 
 - Calculate the number of unhealthy targets.
 - Compare current number against `maxUnhealthy` threshold and temporary short circuits remediation if the threshold is met.
-- Trigger remediation for unhealthy targets i.e request machines for deletion.
+- Trigger remediation for unhealthy targets i.e apply the unhealthy annotation.
 
 Out of band:
-- The owning controller e.g machineSet controller reconciles to meet number of replicas and start the process to bring up a new machine/node.
+- The owning controller e.g machineSet observes the annotation and requests the machine for deletion.
```

Comment:
```suggestion
- The owning controller observes the annotation and is responsible to remediate the machine.
```

Reply:
That sounds good to me. I was thinking maybe the annotation should have the strategy spelled out, something like: I have a health check, I expect this to be deleted.